### PR TITLE
added issue template

### LINKED ISSUE #727: Add issue templates
Add issue templates (bug report, feature requests, etc...).
We can do something similar to kyverno/kyverno-json.

### PR DESCRIPTION
**Pull request by swastik959**
Fixes #727

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: "Bug Report"
+description: Report a bug in the Kyverno JSON.
```
Comment (eddycharly) [inline .github/ISSUE_TEMPLATE/bug.yaml:2]: No Kyverno JSON project here :)

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      value: |
+        Please tell us about the bug.
+  - type: dropdown
+    id: chainsaw-version
+    attributes:
+      label: chainsaw version Version
+      description: >-
+        What version of the chainsaw are you running?
+      options:
+        - 0.1.0
```
Comment (eddycharly) [inline .github/ISSUE_TEMPLATE/bug.yaml:17]:
```suggestion
        - v0.1.0
        - v0.1.1
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      # value: "asdf"
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: >-
+        Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+        chainsaw logs may be found by passing the -v flag to any command.
```
Comment (eddycharly) [inline .github/ISSUE_TEMPLATE/bug.yaml:68]: Not relevant

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      # placeholder: Tell us what you see!
+      # value: "asdf"
+    validations:
+      required: true
+  - type: textarea
+    id: bug-reproduce-steps
+    attributes:
+      label: Steps to reproduce
+      description: >-
+        What are the exact steps needed to reproduce the bug you experienced? Please provide any and all manifests needed to reproduce your issue, including Kyverno policies and test resources.
```
Comment (eddycharly) [inline .github/ISSUE_TEMPLATE/bug.yaml:34]: Not relevant to talk about kyverno policies

**Comment by eddycharly** [comment]
@swastik959 i added a few comments

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 3 above)
Comment (swastik959) [reply .github/ISSUE_TEMPLATE/bug.yaml:68]: so should there be log output or not

Comment (eddycharly) [reply .github/ISSUE_TEMPLATE/bug.yaml:68]: I mean the kyverno policies sentence

**Pull request merged**